src/bitcoin-wallet.cpp

```diff
@@ -31,6 +31,7 @@ static void SetupWalletToolArgs(ArgsManager& argsman)
 
     // Hidden
     argsman.AddArg("salvage", "Attempt to recover private keys from a corrupt wallet", ArgsManager::ALLOW_ANY, OptionsCategory::COMMANDS);
+    argsman.AddArg("wipetxes", "Wipe all transactions from a wallet", ArgsManager::ALLOW_ANY, OptionsCategory::COMMANDS);
```

Comment:
should this be hidden? (is it actually?)

Reply:
It's not and shouldn't be. It looks like we added  "salvage" option in the wrong place https://github.com/dashpay/dash/commit/07a4d482494d29f7cf687ee94a0590a5a6c736d8#diff-0d04b20c6ef91f3f8e0ab7cf0e17dde5665b82e9c838f8ad94fc5bd488757d76R34 and then we failed to remove `// Hidden` https://github.com/dashpay/dash/commit/1253e6d0a9150b16461ede80c9a890e0571a0925#diff-0d04b20c6ef91f3f8e0ab7cf0e17dde5665b82e9c838f8ad94fc5bd488757d76R32 later.